invisible_cities/core/core_functions.py

```diff
@@ -77,6 +83,61 @@ def in_range(data, minval=-np.inf, maxval=np.inf, left_closed=True, right_closed
     return lower_bound & upper_bound
 
 
+def all_in_range(data         : np.ndarray                                   ,
+                 minval       : float                                        ,
+                 maxval       : float                                        ,
+                 display_name : Optional[str] = ''                           ,
+                 strictness   : Optional[Strictness] = Strictness.raise_error,
+                 **kwargs)->bool:
+    """
+    Checks whether input values are all inside the interval (minval, maxval).
+
+    Parameters
+    ----------
+    data : np.ndarray
+        Input array to check.
+    minval: float
+        Lower limit of the interval.
+    maxval: float
+        Upper limit of the interval.
+    display_name: string
+        Label to be displayed in case of warning or exception.
+    strictness: Strictness
+        Describes the behaviour when the output is `False`. If `strictness` is:
+        - `Strictness.raise_error`: an exception is raised.
+        - `Strictness.warning`: a warning is raised before returning.
+        - `Strictness.silent`: the function returns quietly.
+
+    **kwargs:
+        Optional arguments being passed to `in_range`.
+    Returns
```

Comment:
```suggestion
    
    Returns
```
add blank line before Returns